Client/mods/deathmatch/logic/luadefs/CLuaDrawingDefs.cpp

```diff
@@ -1705,6 +1709,11 @@ int CLuaDrawingDefs::DxGetStatus(lua_State* luaVM)
         lua_pushstring(luaVM, "SettingHighDetailPeds");
         lua_pushboolean(luaVM, dxStatus.settings.bHighDetailPeds);
         lua_settable(luaVM, -3);
+
+        lua_pushstring(luaVM, "TotalPhysicalMemory");
+        lua_pushnumber(luaVM, (lua_Number)(SharedUtil::GetWMITotalPhysicalMemory() / 1024 / 1024));
```

Comment:
```suggestion
        lua_pushnumber(luaVM, static_cast<lua_Number>(SharedUtil::GetWMITotalPhysicalMemory()) / 1024.0 / 1024.0);
```
`SharedUtil::GetWMITotalPhysicalMemory()` returns a `long long` integer value, you must promote it to a double (lua_Number) before division.

Reply:
AFAIK if either operand of an arithmetic operation is of floating-point type both operands are converted to be of the same type.
So no manual casts are required.